includes/class-wc-payments.php

```diff
@@ -475,6 +475,8 @@ public static function init() {
 			self::$wc_payments_checkout = new WC_Payments_Checkout( self::$legacy_card_gateway, self::$platform_checkout_util, self::$account, self::$customer_service );
 		}
 
+		self::$card_gateway->maybe_init_subscriptions();
```

Comment:
I think this fixes the issue, but is there a way we can check if this has loaded inside of `maybe_init_subscriptions()`?